Makefile

```diff
@@ -73,23 +73,23 @@ help: ## Display this help.
 ##@ Binary
 
 HELM = $(shell pwd)/bin/helm
-helm: ## Download helm locally if necessary.
+helm-install: ## Download helm locally if necessary.
 	$(call go-install-tool,$(HELM),helm.sh/helm/v3/cmd/helm@v3.9.0)
 
 GINKGO = $(shell pwd)/bin/ginkgo
-ginkgo: ## Download ginkgo locally if necessary.
+ginkgo-install: ## Download ginkgo locally if necessary.
 	$(call go-install-tool,$(GINKGO),github.com/onsi/ginkgo/ginkgo@v1.16.5)
 
 KIND = $(shell pwd)/bin/kind
-kind: ## Download kind locally if necessary.
+kind-install: ## Download kind locally if necessary.
 	$(call go-install-tool,$(KIND),sigs.k8s.io/kind/cmd/kind@v0.14.0)
 
 CONTROLLER_GEN = $(shell pwd)/bin/controller-gen
-controller-gen: ## Download controller-gen locally if necessary.
+controller-gen-install: ## Download controller-gen locally if necessary.
```

Comment:
I added the `-install` because I was having problem with the `helm` task. Seemed as if some other task was overriding that one.
Can we use a different naming convention for these?